versionist.conf.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const execSync = require('child_process').execSync;
+
+const getAuthor = (commitHash) => {
+  return execSync(`git show --quiet --format="%an" ${commitHash}`, {
+    encoding: 'utf8'
+  }).replace('\n', '');
+};
+
+const isIncrementalCommit = (changeType) => {
+  return Boolean(changeType) && changeType.trim().toLowerCase() !== 'none';
+};
+
+module.exports = {
+  // This setup allows the editing and parsing of footer tags to get version and type information,
+  // as well as ensuring tags of the type 'v<major>.<minor>.<patch>' are used.
+  // It increments in a semver compatible fashion and allows the updating of NPM package info.
```

Comment:
minor nit, but perhaps the `NPM` in the line above should be changed to `Python`? ;-)